src/engine/plugins/tile.cpp

```diff
@@ -527,13 +527,12 @@ Status TilePlugin::HandleRequest(const api::TileParameters &parameters, std::str
                     // This is the geometry leading away from the intersection
                     // (i.e. the geometry of the target edge-based-node)
                     second_geometry.clear();
-                    facade.GetUncompressedGeometry(
-                        edge_based_node_info.at(target_ebn).packed_geometry_id, second_geometry);
+                    second_geometry = facade.GetUncompressedReverseGeometry(
```

Comment:
Again we only use the reverse geometry here to extract the missing node that is sliced off the forward geometry. We could remove this if we change the interface.